tests/tests_deployment/test_conda_store_scaling.py

```diff
@@ -0,0 +1,234 @@
+import base64
+import json
+import logging
+import os
+import sys
+import time
+import uuid
+
+import pytest
+import requests
+from kubernetes import client, config, dynamic
+
+from tests.tests_deployment import constants
+
+CONDA_STORE_API_ENDPOINT = "conda-store/api/v1"
+NEBARI_HOSTNAME = constants.NEBARI_HOSTNAME
+NAMESPACE = os.getenv("CONDA_STORE_SERVICE_NAMESPACE")
+TEST_CONDASTORE_WOKER_COUNT = os.getenv("TEST_CONDASTORE_WOKER_COUNT", 1)
+# NEBARI_HOSTNAME = "local.quansight.dev" ## Override for local testing
```

Comment:
Seems like this line should be removed, right?

Reply:
yes.